Dockerfile

```diff
@@ -15,13 +15,10 @@ RUN apk add git
 WORKDIR /code
 COPY . /code/
 # See https://github.com/CosmWasm/wasmvm/releases
-ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.5.2/libwasmvm_muslc.aarch64.a /lib/libwasmvm_muslc.aarch64.a
-ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.5.2/libwasmvm_muslc.x86_64.a /lib/libwasmvm_muslc.x86_64.a
-RUN sha256sum /lib/libwasmvm_muslc.aarch64.a | grep e78b224c15964817a3b75a40e59882b4d0e06fd055b39514d61646689cef8c6e
-RUN sha256sum /lib/libwasmvm_muslc.x86_64.a | grep e660a38efb2930b34ee6f6b0bb12730adccb040b6ab701b8f82f34453a426ae7
-
-# Copy the library you want to the final location that will be found by the linker flag `-lwasmvm_muslc`
-RUN cp /lib/libwasmvm_muslc.${arch}.a /lib/libwasmvm_muslc.a
```

Comment:
is this not needed anymore?

Reply:
The new version of wasmvm now links against `wasmvm_muslc.x86_64` / `wasmvm_muslc.aarch64` instead of `wasmvm_muslc`, so the renaming is not needed anymore.
It was changed in this PR: https://github.com/CosmWasm/wasmvm/pull/514/files

Reply:
Chris, could you add this change to https://github.com/CosmWasm/wasmvm/blob/main/docs/MIGRATING.md?